css/styles.css

```diff
@@ -0,0 +1,376 @@
+    font-family: 'Ubuntu', sans-serif;
+}
+
+.title_container {
+    width: 100%;
+    display: flex;
+    align-self: center;
+    padding-left: 30px;
+    padding-bottom: 150px;
+}
+
+.title_container .inner_container {
+    display: flex;
+    flex-direction: column;
+    padding-left: 40vw;
+}
+
+h2 {
+    color: #f5f5f5;
+    font-size: 4vw;
+}
+
+nav p {
+    color: #f5f5f5;
+    font-size: 1.5vw;
+    margin-top: 1vw;
+}
+
+nav .container {
+    display: flex;
+    position: fixed;
+}
+
+nav .container #stickybox {
+    background-color: #f5f5f5;
+    margin-left: 6vw;
+    width: 2vw;
+    height: 25vw;
+    border-radius: 0 0 20px 20px;
+}
+
+nav .drop_container {
+    position: fixed;
+}
+
+nav .drop {
+    background-color: #f5f5f5;
+    width: 2vw;
+    height: 4vw;
+    border-radius: 20px;
+    position: relative;
+    animation-name: drip;
+    animation-duration: 4.5s;
+    animation-iteration-count: infinite;
+}
+
+@keyframes drip {
+    0% {
+        left: 6vw; top: 18vw;
+    }
+    100% {
+        left: 6vw; top: 80vw;
+    }
+}
+
+nav .container #titlebox {
+    margin-left: 1vw;
+    display: flex;
+    flex-direction: column;
+}
+
+nav .icon {
+    padding: 1.5vw 1vw;
+    width: 2.5vw;
+    height: auto;   
+}
+
+@media screen and (max-width: 450px) {
+    h2 {
+        font-size: 8vw;
+    }
+
+    nav p {
+        font-size: 3vw;
+        margin-top: 2vw;
+    }
+
+    nav .icon {
+        padding: 3.4vw 1vw;
+    }
+
+    nav .container #stickybox {
+        height: 40vw;
+    }
+    
+    nav .drop {
+        animation-duration: 6s;
+    }
+    @keyframes drip {
+        0% {
+            left: 6vw; top: 18vw;
+        }
+        100% {
+            left: 6vw; top: 250vw;
+        }
+    }
+}
+
+@media screen and (max-width: 800px) and (min-width: 460px) {
+    @keyframes drip {
+        0% {
+            left: 6vw; top: 18vw;
+        }
+        100% {
+            left: 6vw; top: 120vw;
+        }
+    }
+}
+
+/* section */
+section {
+    display: flex;
+    justify-content: center;
+    background-color: #3fc1c9;
+    width: 100%;
+    height: 52vw;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+section .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 70%;
+}
+
+h1 {
+    color: #f5f5f5;
+    margin: 2vw;
+    font-size: 3vw;
+}
+
+section .row {
+    display: flex;
+    justify-content: space-evenly;
+    width: 100%;
+}
+
+section .column {
+    background-color: #f5f5f5;
+    height: 20vw;
+    width: 100%;
+    margin: 15px;
+    padding: 15px;
+    font-size: 1.5vw;
+    border-radius: 1vw;
+}
+
+section #top_column {
+    height: 8vw;
+}
+
+@media screen and (max-width: 450px) {
+    section {
+        height: 105vw;
+    }
+
+    section .row {
+        display: flex;
+        flex-direction: column;
+        font-size:
+    }
+
+    section #top_column {
+        height: 60px;
+    }
+
+    section .column {
+        width: 100%;
```

Comment:
careful with this, looks like it makes the columns spill out of the right side of the container on mobile. Removing it does cause some issues with the text.